Gemfile

```diff
@@ -0,0 +1,10 @@
+source "http://rubygems.org"
+
+group :development do
+  gem 'therubyracer'
+  gem 'jekyll'
+  gem 'kramdown'
+  gem 'pygments.rb'
+  gem 'rake'
+  gem 'html-proofer'
+end
```

Comment:
Ah ok, that makes sense...

I'll try to figure out what travis is doing then that bundler is complaining, maybe there's nothing wrong here.